### DISCUSSION TITLE
legacy branch

### DISCUSSION THREAD

**Discussion by xyzbtw** [3arthqu4ke/3arthh4ck#210] [Q&A] [answered]
Where did the legacy branch go? 😢

**Comment by 3arthqu4ke** [comment, marked as answer]
Just browse the repository at this commit https://github.com/3arthqu4ke/3arthh4ck/tree/0a8c90f0e5f4d100faf48f22bd9a187b2b4be705 I think, maybe later. Nothing anyone would need it for anyways.

**Reply by xyzbtw** [reply]
Thanks, i needed it to test something
